Clients/src/presentation/pages/ProjectView/ProjectSettings/index.tsx

```
@@ -197,29 +200,34 @@ const ProjectSettings: FC<ProjectSettingsProps> = React.memo(
 
     const handleConfirmDelete = useCallback(async () => {
       try {
-        // TO DO: Make a call to with dynamic project id
-        const projectId = 1;
         const response = await deleteEntityById({ routeUrl: `/projects/${projectId}` });
+        debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)